src/Hero/index.js

```diff
@@ -0,0 +1,23 @@
+import App from "../App";
```

Comment:
I think this can be removed as its greyed out in PHP storm, sorry I missed this the first time!